flink-end-to-end-tests/test-scripts/common_mesos_docker.sh

```diff
@@ -25,6 +25,8 @@ source "$(dirname "$0")"/common_docker.sh
 MAX_RETRY_SECONDS=120
 IMAGE_BUILD_RETRIES=5
 NODENAME=${NODENAME:-`hostname -f`}
+export MESOS_FLINK_DIR="/flink"
```

Comment:
Do not need to export. `common_mesos_docker.sh` will be `source` in the mesos related tests.